nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1426,6 +1426,22 @@
                 sys.exit("Nuitka: For this environment to load, need to use this as an option to compile with '--include-module=%s'." % mod_name)
       when: 'not deployment'
 
+- module-name: 'h2o' # checksum: a246b3c9
+  data-files:
+    dirs:
+      - 'h2o_data/'
+    patterns:
+      - '*.txt'
+      - '*.csv'
+  dlls:
+    - from_filenames:
+        relative_path: 'backend/bin/'
+        prefixes:
+          - 'h2o'
+        suffixes:
+          - 'jar'
+        executable: 'yes'
```

Comment:
Are jar files really executables, or should this not be data files too?

Reply:
do you consider `.py` files executables? if you you'd execute one via python helloworld.py right?
for `jar` files, you'd do the same `java -jar helloworld.jar`
there's also the fact that jar files can have an manifest file containing the main entry point and those sort of jars are considered executable, i think h2o executes a subprocess to run this file, so it should be considered an executable.

Reply:
A jar file is not an executable by itself in my book, just because it is input to the `java` executable, to Nuitka is a data file totally, since we are not going to do anything but copying it and maybe embedding it. But I was wondering if the "executable" flag means that e.g. on Linux, an x-bit would be required, like if `helloworld.jar` would be executable somehow, and since these are zip files, I imagine they can `actually` have a runner in front, in which case, they actually are executables. That runner would have to be made portable, signed, etc potentially.

I think Python files are too messy as an example to discuss. I believe they are never data files, but currently we do not have the deployment options that would compile them to Python code for inclusion. That really must change, since providing editable plugins is a valid use case, but it doesn't get done so far.